### ISSUE TITLE
[MAS2.4.1][Screen reader-Welcome] Landmarks are not defined for Bot emulator application except banner landmark 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1833]
**Actual Result:**
Landmarks are not defined for Bot emulator application except banner landmark 

**Expected Result:**
Landmarks should be defined for Bot emulator application 
Main Landmark, Navigation Landmark, Header Landmark …etc 

Note: This issue exits throughout the application

**Repro Steps:**
Turn on the NVDA
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the Welcome screen  
3.	Navigate to the landmark’s mode using ‘D; key 
4.	Observe the issue

**User Impact:**
Users who rely on screen reader will be affected if landmarks are not defined for application then the screen reader users will not able to navigate desired location 

**Test Environment:**
OS Build: Version 1903(OS Build 18362.295)
Application: Bot Framework V4 Emulator
Screen reader: NVDA 

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d} 
[MAS2.4.1_Landmarks.zip](https://github.com/microsoft/BotFramework-Emulator/files/3578716/MAS2.4.1_Landmarks.zip)

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 with NVDA(2019.2) issue got fixed hence we are closing this bug.
Now NVDA is reading the Navigation ,Main and Primary region Landmarks
